Doc/c-api/sys.rst

```diff
@@ -216,6 +216,37 @@ Operating System Utilities
       The function now uses the UTF-8 encoding on Windows if
       :c:member:`PyPreConfig.legacy_windows_fs_encoding` is zero.
 
+.. c:function:: FILE* Py_fopen(PyObject *path, const char *mode)
+
+   Similar to the :c:func:`!fopen` function, but *path* is a Python object and
+   an exception is set on error.
+
+   *path* must be a :class:`str` object or a :class:`bytes` object.
```

Comment:
It can also be a path-like object, isn't? But its support is inconsistent on Windows.